python/tests/catalog/test_hive.py

```diff
@@ -468,7 +468,7 @@ def test_list_namespaces():
 
 
 def test_list_namespaces_with_parent():
-    catalog = HiveCatalog(HIVE_CATALOG_NAME, {}, uri=HIVE_METASTORE_FAKE_URL)
+    catalog = HiveCatalog(name=HIVE_CATALOG_NAME, properties={}, uri=HIVE_METASTORE_FAKE_URL)
```

Comment:
We can just omit properties since we just now use the `kwargs` as the properties. This makes it a bit more flexible to configure the different catalogs

Reply:
```suggestion
    catalog = HiveCatalog(HIVE_CATALOG_NAME, uri=HIVE_METASTORE_FAKE_URL)
```